query/src/org/labkey/query/controllers/QueryController.java

```
@@ -6685,6 +6685,28 @@ public Object execute(Object o, BindException errors) throws Exception
     @RequiresPermission(ReadPermission.class)
     public class GetQueryEditorMetadataAction extends ReadOnlyApiAction<QueryForm>
     {
+        @Override
+        protected ObjectMapper createRequestObjectMapper()
+        {
+            PropertyService propertyService = PropertyService.get();
+            if (null != propertyService)
+            {
+                ObjectMapper mapper = JsonUtil.DEFAULT_MAPPER.copy();
+                propertyService.configureObjectMapper(mapper, null);
+                return mapper;
+            }
+            else
+            {
+                return null;

```

Probably better to throw an exception here, because returning null will likely cause a NullPointerException wherever it's used which will be disconnected from the place that detected the problem.